### DISCUSSION TITLE
在同一台设备两个不同浏览器共享一份简悦（稍后读数据）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3223] [使用技巧]
> 同一台设备，在 A 浏览器添加稍后读后在 B 浏览器查看。

https://user-images.githubusercontent.com/81074/146911765-4374ee03-c301-4de5-b563-6917be35ece8.mp4

## 使用场景

如果你每天都在长时间使用浏览器，有没有想过，通过将工作与私人分开的方式？这样方便切换工作与私人的不同环境，使用简悦很容易实现这个需求。

在工作（主）浏览器保存的稍后读，（使用简悦）在私人（副）浏览器马上看到。

## 使用前提

1. 简悦 [高级账户](https://simpread.pro/price)
2. 配置简悦 [自动同步](http://ksria.com/simpread/docs/#/自动同步)

## 如何配置

因为配置了自动同步，所以你在本地肯定有一份 `simpread_config.json`，所以在副浏览器，只需要：

1. 安装简悦
2. 选项页 → 共通 → 从本地导入配置，然后找到你在本地的 `simpread_config.json`，导入后简悦自动会帮你做好所有的事情。

![image](https://user-images.githubusercontent.com/81074/146914910-5d4e07dc-03ed-4dc8-a44e-be3b70f627d3.png)

就这么简单。

## 关联

1.  [自动同步](http://ksria.com/simpread/docs/#/自动同步)
2. [知识库](https://github.com/Kenshin/simpread/discussions/2221)

**Comment by andrewcui512** [comment]
学到了，这回彻底从 Chrome 迁移到 Vivaldi了，连夜居家搬走

**Reply by Kenshin** [reply]
哈哈，工作与私人分开的方式还是挺好的，我已经尝试了一小段时间。
